.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: node_js
+
+node_js:
+  - "lts/*"
```

Comment:
This does not work as you'd think. Can you please put Node 8 there?

Reply:
I've followed [this](https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#Specifying-Node.js-versions) guide.

Reply:
That guide is not correct and it does not produce stable results. When we tag a new lts major version in Node.js, the CI will stop testing on the previous LTS and start testing the new.